test/Proxy.t.sol

```diff
@@ -23,8 +23,8 @@ contract ProxyUpgradeTest is Test {
     IRegistry registry;
     IChamber chamber;
 
-    IChamberProxy registryProxy;
-    IChamberProxy chamberProxy;
+    IChamberProxy multiProxy;
+    IChamberProxy MultiProxy;
```

Comment:
This is confusing. Please revert back to calling the proxies `registryProxy` and `chamberProxy`

Reply:
done